apiary.apib

```diff
@@ -3067,6 +3084,37 @@ Returns navigation information
         }
 
 
+## Course Certificates Status [/courses/{course_id}/certificates/status]
+
+It returns status for course certificates
+
+### Course Certificates Status [GET]
+
+Returns a JSON representation of certificate details for a specified course, also return status if we can request certificate generation for the course, possible statuses can be 'available', 'notavailable', 'generated', 'generating'
```

Comment:
Our use case is to display a button to apros admin if course certificates are available i-e course end date or any other criteria met if any and change the status of the button accordingly. Apros admin would click this button to generate certificates for all participants for the specified course.